### DISCUSSION TITLE
trace rust-flake log messages

### DISCUSSION THREAD

**Discussion by dezren39** [juspay/omnix#230] [Q&A] [answered]
hey so i get these messages but i don't know why, i didn't have any traces before adding the omnix flake, is there a way to hide these if they aren't an error? or if they are an error then what exactly do i need to do?
```
┏━ 6 Traces:
 ⋮
┃ trace: rust-flake: Using /nix/store/7zckygczcw7rcvygjrhagg8jkiy0gpvv-nix_health/crate.nix
┃ trace: rust-flake: Using /nix/store/6nyn5bia9hpz8y9sj9da0g6qp3ci7g29-nix_rs/crate.nix
┃ trace: rust-flake: Using /nix/store/fq3d54yl2pfz5dvzn442h947r9l6rg0i-nixci/crate.nix
┃ trace: rust-flake: Using /nix/store/yhajwg1sjf6bvlxfcscxwm9sbzq9g3xc-omnix-cli/crate.nix
┃ trace: rust-flake: Using /nix/store/lm9bqn4jhi7bfxix9mpqkzvqgd7zccr0-omnix-gui/crate.nix
┣━━━
┗━ ∑ ⚠ Exited with 6 traces reported by nix at 03:34:52 after 27s
```

**Comment by shivaraj-bh** [comment, marked as answer]
rust-flake uses `builtins.traceVerbose` for logging the location of `.nix` for each of the crates (See https://github.com/juspay/rust-flake/blob/747daa629bd956c9e14b1436b5f425ac82a01d14/nix/modules/default-crates.nix#L38). 

You can remove the `--trace-verbose` argument from `.envrc` of `omnix` to disable this behaviour.

A bit more about `traceVerbose`: https://nixos.asia/en/traceVerbose
